.github/workflows/form_builder_phone_field.yaml

```diff
@@ -0,0 +1,86 @@
+name: Form Builder Phone Builder
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    tags:
+      - 'form_builder_phone_field-v*'
+    branches: [master]
+    paths:
+      - "packages/form_builder_phone_field/**"
+      - ".github/workflows/form_builder_phone_field.yaml"
+
+  pull_request:
+    branches: [master]
+    paths:
+      - "packages/form_builder_phone_field/**"
+      - ".github/workflows/form_builder_phone_field.yaml"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        channel: [stable, beta]
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Flutter action
+        uses: subosito/flutter-action@v2
+        with:
+          channel: ${{ matrix.channel }}
+
+      - name: Run Tests
+        run: |
+          pushd packages/form_builder_phone_field
+          flutter pub get
+          flutter format --dry-run --set-exit-if-changed .
+          flutter analyze --no-pub
+          flutter test --no-pub --coverage
+
+      - name: Build Example
+        run: |
+          pushd packages/form_builder_phone_field/example
+          flutter build appbundle --no-pub --debug
+          flutter build ios --no-pub --debug --no-codesign
+
+#       - name: Upload coverage to Codecov
+#         if: ${{ matrix.channel == 'stable' }}
+#         uses: codecov/codecov-action@v1
+#         with:
+#           file: coverage/lcov.info
+  deployment:
+    if: ${{ github.ref_type == 'tag' }}
+    needs: build
+    name: Deploy package
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Configure enviroment
+      uses: actions/checkout@v3
+    - name: Download flutter
+      uses: subosito/flutter-action@v2.4.0
+      with:
+        channel: 'stable'
+    - name: Setup pub credentials
+      shell: bash
+      env:
```

Comment:
@danvick please add secrets to test deployment

Can find this keys with [this steps](https://birju.dev/posts/publish-your-flutter-package-using-github-actions/):

> If you have already published your package before, you can easily find the required credentials for [pub.dev](https://pub.dev/) as you already have logged into your account. You can locate the file at the following path:
> 
> Linux/Mac-OS
> `~/.pub-cache/credentials.json`
> Windows
> `C:\Users\<USER>\AppData\Roaming\Pub\Cache\credentials.json`

Reply:
Done